lua/nvim-drawer/init.lua

```diff
@@ -42,6 +42,24 @@ local mod = {}
 --- is open.
 --- Called in the context of the drawer window.
 --- @field on_did_open? fun(event: { instance: DrawerInstance, winid: integer, bufnr: integer }): nil
+--- @field get_win_config? fun(): DrawerWindowConfig
+
+--- Adapted from `vim.api.keyset.win_config`
+--- @class DrawerWindowConfig
+--- @field margin? number
+--- @field width? number
+--- @field height? number
+--- @field anchor? 'NE' | 'NC' | 'NW' | 'CE' | 'CC' | 'CW' | 'SE' | 'SC' | 'SW'
```

Comment:
add `N | E | S | W | C` since they're more ergonomic, but they should just resolve to `NC / CE / SC / CW / CC` since those make the code much cleaner